Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM golang:1.10.3-alpine3.8
+FROM golang:1.10.4-alpine3.8 as build
+WORKDIR /go/src/github.com/securego/gosec
+COPY . .
+RUN apk add -U git make
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN make
 
-ENV BIN=gosec
-ENV GOROOT=/usr/local/go
-ENV GOPATH=/go
-
-COPY $BIN /go/bin/$BIN
-COPY docker-entrypoint.sh /usr/local/bin
-
-ENTRYPOINT ["docker-entrypoint.sh"]
+FROM golang:1.10.4-alpine3.8
+RUN apk add -U gcc musl-dev
```

Comment:
`gosec` does not require `cgo` to operate. It works fine without and the image is `100MB` smaller. Please could you provide a some details about your use case which requires cgo?

Reply:
fixed

Reply:
Without `gcc` and `musl-dev`:
```
$ docker run --rm -t -v `pwd`:/go/src/github.com/opencontainers/runc -w /go/src/github.com/opencontainers/runc securego/gosec -include G401 -quiet ./...
exec: "gcc": executable file not found in $PATH
cgo failed: [go tool cgo -objdir /tmp/github.com_opencontainers_runc_libcontainer_system_C530284581 -- -I /tmp/github.com_opencontainers_runc_libcontainer_system_C530284581 sysconfig.go]: exit status 1
...
```

With `gcc` and `musl-dev`:
```
$ docker run --rm -t -v `pwd`:/go/src/github.com/opencontainers/runc -w /go/src/github.com/opencontainers/runc securego/gosec -include G401 -quiet ./...
```